src/api/app/models/user.rb

```diff
@@ -373,7 +443,7 @@ def authenticate(user_login, password)
       # User account is not confirmed yet
       return if user.try(:state) == STATES['unconfirmed']
 
-      Rails.logger.debug "Authentificated user '#{user.try(:login)}'"
+      Rails.logger.debug "Authentificated user #{user.inspect} '#{user.try(:login)}'"
```

Comment:
If you inspect the whole user object, then don't do `user.try(:login)`. Or remove the inspect if this was some debug logging at development.